server.php

```diff
@@ -30,6 +62,11 @@ function show_valet_404()
     parse_url($_SERVER['REQUEST_URI'], PHP_URL_PATH)
 );
 
+/**
+ * Filter host to support xip.io feature
+ */
+$_SERVER['HTTP_HOST'] = support_xip_io($_SERVER['HTTP_HOST']);
```

Comment:
We need the HTTP_HOST to stay as the full URL including `xip.io` otherwise redirects generated by your application will break, because they need to include `xip.io` in the redirect, but they won't if the HTTP_HOST has been overridden.

So this line can be removed entirely, and instead we should just use the `support_xip_io` function directly on line 71.